### DISCUSSION TITLE
posts/my_blogging_stack/

### DISCUSSION THREAD

**Discussion by giscus** [rasnes/homepage-comments#3] [Announcements]
# posts/my_blogging_stack/

My complete blogging process, from idea generation to deployment, including tools like Logseq, Typora, LanguageTool, Hugo, Netlify, and more.

https://rasmusnes.com/posts/my_blogging_stack/

**Comment by mihaiconstantin** [comment]
I’m currently facing a similar decision, and your insights have been helpful. In addition to what you mentioned in your other blog post, I’ve noticed that there seem to be more resources available for building Hugo themes. Do you have any experience with building a Quarto theme? I’m interested in more than just modifying the `.scss` file—I would like to know what control I have over adjusting the entire layout to my heart's content. Cool blog!

**Reply by rasnes** [reply]
Hi, and thanks for your kind words! Very sorry for the late reply, I hadn't configured notifications for my `giscus` repo properly, so I completely missed your comment.

If still relevant: No, I did not investigate how to customize a Quarto theme. Hugo is more common than Quarto, so it seems only reasonable that there are more resources and documentation for it. I also think you'd get better help from an LLM, like Sonnet-3.5 or R1, when building or tweaking a theme based on Hugo than on Quarto. My two cents on your choice: I would recommend Hugo if you _need_ customization, Quarto blog seems more like a good fit if you want something off-the-shelf that is easy to spin up with minimal hassle.

**Reply by mihaiconstantin** [reply]
Hi! No worries about the late reply. Thanks for writing back. In the end, I used Astro because I found it much faster to get started with (i.e., due to the high-quality documentation). However, I think Hugo would've also been a fine choice. Indeed, it helps quite a bit using these LLMs for tweaking things around.
